roles/repositories/README.md

```diff
@@ -0,0 +1,131 @@
+theforeman.foreman.repositories
+===============================
+
+This role enables Red Hat Repositories, creates Custom Products and Repositories, and defines Sync Plans.
+
+Role Variables
+--------------
+
+This role supports the [Common Role Variables](https://github.com/theforeman/foreman-ansible-modules/blob/develop/README.md#common-role-variables).
+
+- `products`: List of products to enable and synchronize. Each product is represented as a dictionary and can include `repository_sets` which represent Red Hat Repositories and should be used when the product name matches an existing Red Hat Product. Each repository set must have a `name` and should specify the `basearch` and/or `releasever` only when multiple versions are available for that product. Alternatively a custom product can have associated `repositories` which represent custom repositories. Custom repositories are required to have a `url` and `content_type` and may require additional fields. A variety of examples are demonstrated in the data structure below:
```

Comment:
Custom repositories require a name, too, right?

Personal preference: Include a line break after each sentence.

Reply:
Done, but I admit I'm still not fully satisfied with the overall flow and formatting of the doc